src/hotspot/share/utilities/packedTable.hpp

```diff
@@ -22,6 +22,9 @@
  *
  */
 
+#ifndef SHARE_UTILITIES_PACKEDTABLE_HPP
+#define SHARE_UTILITIES_PACKEDTABLE_HPP
+
```

Comment:
I just backported this to JDK 25.  I wonder if you should omit this change and put it under another CR so we can backport that too.  Or not backport this, assuming that JDK 25 will never have double inclusions of this file ?